python/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "uagents"
-version = "0.13.1"
+version = "0.13.2"
```

Comment:
You might want to revert this change so that you have something to commit for the release PR from branch `release/v0.13.2`.

Reply:
You're right, thanks. I'll create a separate PR for that :)